src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -568,7 +568,7 @@ public void ValidatePermissionsInConfig(RuntimeConfig runtimeConfig)
                                 }
                             }
 
-                            if (!IsValidDatabasePolicyForAction(configOperation))
+                            if (runtimeConfig.DatabaseType is not DatabaseType.mssql && !IsValidDatabasePolicyForAction(configOperation))
```

Comment:
in pr title, add that this is mssql specific and then where you note that in body, could you reference the issue numbers for pgsql and mysql implementation of this feature?